internal/pkg/service/stream/storage/level/local/volume/diskalloc/allocator_linux.go

```diff
@@ -31,5 +31,6 @@ func Allocated(path string) (datasize.ByteSize, error) {
 	}
 
 	// Note: Blksize is in bits not bytes
-	return datasize.ByteSize((sysStat.Blksize / 8) * sysStat.Blocks), nil
+	// Note: using type int64 for arm64 (MacOS)
+	return datasize.ByteSize((int64(sysStat.Blksize / 8)) * int64(sysStat.Blocks)), nil // nolint:unconvert
```

Comment:
- The original comment should be kept.
- The new comment is not accurate.

My suggestion:
```
// Notes:
// 1. "/ 8": Blksize is in bits not bytes
// 2. The type of fields can vary depending on the architecture (int32/int64), so retyping it to int64 is necessary.
```

Reply:
Please, also send a PR with the fixed `.air.toml`, as we discussed.